tests/sqlparser_common.rs

```diff
@@ -2492,26 +2473,6 @@ fn parse_derived_tables() {
             }],
         }))
     );
-
-    // Nesting a subquery in parentheses is non-standard, but supported in Snowflake SQL
-    let sql = "SELECT * FROM ((SELECT 1) AS t)";
```

Comment:
To reiterate, it's hard to review when you change code at the same time you move it around. I forgot about this when reviewing the PR for the new dialect, but the idea was to do the moves there.

I found the new location of the other two moved tests, but not this one.. Where was this test case moved?

Reply:
Sorry about that , i also comment in the [previous pr](https://github.com/ballista-compute/sqlparser-rs/pull/259#issuecomment-671435447) that i don't sure what needed to be in the test (now i am understand :) ) . 

Anyway added also this test ..